src/main/createWindow.js

```diff
@@ -69,11 +69,22 @@ const createWindow = (pathname, options = {}) => {
       app.addRecentDocument(pathname)
       const filename = path.basename(pathname)
       fs.readFile(path.resolve(pathname), 'utf-8', (err, file) => {
```

Comment:
We have assigned the `encode` of `utf-8`. So do we still need to add UTF-8 BOM? or just windows needed?

Reply:
No, we don't need to change anything. Node.js doesn't support UTF-8 BOM. So I created `getMarkdownWithEncoding(...)` which adds the BOM sequence `\uFEFF` (`EF BB BF`) before save to the markdown content.

The BOM sequence is just a text which indicates the file as UTF-8 encoded.

E.g:

```
EF BB BF 61 (HEX)
^^^^^^^^ | --->

^^^ := BOM sequence
after | := text
```

The extracted text from the "file" above is `a` (Hex 61).